server/routes/meetups.js

```diff
@@ -12,12 +12,13 @@ import rsvpController from '../controller/rsvp';
 import Auth from '../middleware/Auth';
 import meetupValidation from '../middleware/validatemeetups';
 import tryCatch from '../utilities/trycatch';
+import validateId from '../middleware/validateid';
 
 router.post('/', Auth.verifyToken, Auth.adminAuth, meetupValidation.validCreateMeetup, meetupValidation.checkDate, tryCatch(meetupController.createMeetup));
 router.get('/upcoming/', Auth.verifyToken, tryCatch(meetupController.getUpcomingMeetups));
-router.get('/:id', Auth.verifyToken, tryCatch(meetupController.getSingleMeetup));
+router.get('/:id', Auth.verifyToken, validateId, tryCatch(meetupController.getSingleMeetup));
 router.get('/', Auth.verifyToken, tryCatch(meetupController.getAllMeetups));
-router.patch('/:meetupId/rsvps', Auth.verifyToken, tryCatch(rsvpController.respondToRsvp));
-router.delete('/:id', Auth.verifyToken, Auth.adminAuth, tryCatch(meetupController.deleteMeetup));
+router.patch('/:meetupId/rsvps', Auth.verifyToken, validateId, tryCatch(rsvpController.respondToRsvp));
+router.delete('/:id', Auth.verifyToken, Auth.adminAuth, validateId, tryCatch(meetupController.deleteMeetup));
```

Comment:
Since `Auth.verifyToken` is applied to all the routes here, you can move it to the index file to remove the repetition.